packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -868,6 +868,72 @@ const defaultConfigs = {
       ETHBTC_PERP: { type: "uniswap", invertPrice: true, uniswapAddress: "0x899a45ee5a03d8cc57447157a17ce4ea4745b199" },
       PERP_FRM: { type: "frm", perpetualAddress: "0x32f0405834c4b50be53199628c45603cea3a28aa" }
     }
+  },
+  BALUSD: {
+    type: "expression",
+    expression: `
+      BALANCER_ETH_INVERTED = 1 / SPOT_BALANCER_ETH;
+      SPOT_BALANCER = BALANCER_ETH_INVERTED * ETHUSD;
+      median(SPOT_BINANCE, SPOT_COINBASE_PRO, SPOT_BALANCER)
```

Comment:
Cool example of skipping the medianizer and just using the ExpressionPriceFeed. @mrice32 do you think this is a better approach than using the Medianizer?

Reply:
Yes, I do. I would be in favor of deprecating the medianizer and eventually removing it in favor of just using the expression price feed. Same with the basket spread price feed.